src/corehost/cli/deps_entry.cpp

```diff
@@ -104,8 +104,8 @@ bool deps_entry_t::to_full_path(const pal::string_t& base, pal::string_t* str) c
     }
 
     pal::string_t new_base = base;
```

Comment:
The lock file preserves all casing, except in one case: `"path"` property under a package library will go from `Newtonsoft.Json/8.0.3` to `newtonsoft.json/8.0.3`. This is most apparent in the assets contains in a package. For example, the string `lib/net45/Newtonsoft.Json.dll` will remain `lib/net45/Newtonsoft.Json.dll`.